llpc/context/llpcCompiler.cpp

```diff
@@ -889,15 +887,9 @@ Result Compiler::BuildPipelineWithRelocatableElf(
         ShaderEntryState cacheEntryState  = ShaderEntryState::New;
         BinaryData elfBin = {};
 
-#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION < 38
-        constexpr uint32_t ShaderCacheCount = 2;
-        ShaderCache*     shaderCache[ShaderCacheCount]  = { nullptr, nullptr };
-        CacheEntryHandle hEntry[ShaderCacheCount]        = { nullptr, nullptr };
-        cacheEntryState = LookUpShaderCaches(pUserShaderCache, &cacheHash, &elfBin, shaderCache, hEntry);
-#else
-        CacheEntryHandle hEntry = nullptr;
-        cacheEntryState = LookUpShaderCache(&cacheHash, &elfBin, &hEntry);
-#endif
+        ShaderCache* shaderCache;
+        CacheEntryHandle hEntry;
```

Comment:
hEntry -> hEntry = nullptr;

Reply:
I maintain that in cases like this one, where the variables are *immediately* used as out-variables for a following function call in all paths, it is more helpful for the understanding of the program to not initialize the variables like this.